fieldservice_agreement/views/agreement_view.xml

```diff
@@ -24,7 +24,7 @@
                     type="object"
                     class="oe_stat_button"
                     icon="fa-laptop"
-                    context="{'default_agreement_id': active_id}"
+                    context="{'default_agreement_id': id}"
```

Comment:
why not `active_id`?

Reply:
I got from here : 

> If you are using active_id, active_ids or active_model in your views (for example, for a smart-button context), you should replace them by the corresponding good practices:
> 
>     id instead of active_id.
>     Hard-coded model name instead of active_model.
>     active_ids doesn't have direct replacement candidate. It will depend on what you want to achieve. It's a weird case though.

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0